lib/collection/src/operations/types.rs

```diff
@@ -155,6 +155,17 @@ pub struct SearchRequest {
     pub with_vector: Option<bool>,
 }
 
+#[derive(Debug, Deserialize, Serialize, JsonSchema)]
+#[serde(rename_all = "snake_case")]
+pub struct PointRequest {
+    /// Look for points with ids
+    pub ids: Vec<PointIdType>,
+    /// Payload interface
+    pub with_payload: Option<WithPayloadInterface>,
+    /// Return point vector with the result. Default: false
+    pub with_vector: Option<bool>,
```

Comment:
If default is `false` - why not just make it a `bool` instead of `Option<bool>`?
You can use [this](https://serde.rs/field-attrs.html#default) serde attribute.

Reply:
TBH I just moved and documented the code from https://github.com/qdrant/qdrant/pull/343/files#diff-90a720d60aa41e3d5ae04a69bc550bef06ff0b7347496e97ff1392cc1e604a12L16

I will look into the serde field attribute to check if I can simplify this :+1:

Reply:
I see, ok

Reply:
I would have to change `SearchRequest` & `ScrollRequest` as well for consistency and propagate the changes.
I'd rather keep it that way for the time being and refactor all requests together when necessary.

WDYT?

Reply:
Ok, maybe let's create an issue for this? @generall what do you think?

Reply:
`Optional` is required to avoid forcing user to specify it. If it can be changed without interrupting openAPI specification - let's change.

Reply:
As far as I researched - default values are not supported in serde without some special tricks like `try_from`

Reply:
I think it can be easily done with https://serde.rs/field-attrs.html#default as `false` is the default value for `bool`

Reply:
If we wanted `true` there we could use https://serde.rs/field-attrs.html#default--path to supply value

Reply:
I can push a commit with the change so we can see if it works and if we like it :)

Reply:
Here is the openAPI spec change https://github.com/qdrant/qdrant/pull/343/commits/24ce24b0bced177cf8795a5e2f3528c3e82a88be

It seems the default value is properly propagated